laser_encoders/language_list.py

```diff
@@ -13,6 +13,34 @@
 # -------------------------------------------------------
 # Language mapping to handle different language codes and names
 
+
+def build_language_names_dict(language_list, language_names):
```

Comment:
Can you please write a short readme for this function and annotate the input and output types?
Currently, the code is difficult to understand for anyone without our context.